src/xmp-tags.js

```diff
@@ -54,20 +75,20 @@ function readTags(tags, chunkDataView) {
         tags._raw = (tags._raw || '') + raw;
         const rdf = getRDF(doc);
 
-        return objectAssign(tags, parseXMPObject(convertToObject(rdf, true)));
+        objectAssign(tags, parseXMPObject(convertToObject(rdf, true)));
     } catch (error) {
-        return tags;
+        return true;
     }
+    return false;
```

Comment:
That looks a bit weird for my taste, a bit like bash, where exit code 0 is marking a success...

Reply:
Yeah, it didn't feel perfect. It made the code simpler than the first version that used exceptions but I will give it some more thought.

Reply:
I just flipped it around.